docs/modules/ROOT/pages/cops_performance.adoc

```diff
@@ -405,6 +405,46 @@ array.min_by(&:foo)
 array.sort_by { |a| a[:foo] }
 ----
 
+== Performance/ConstantRegexp
+
+|===
+| Enabled by default | Safe | Supports autocorrection | VersionAdded | VersionChanged
+
+| Pending
+| Yes
+| Yes
+| 1.9
+| -
+|===
+
+This cop finds regular expressions whose all dynamic components are constants.
```

Comment:
Just noticed this... Better description: " This cop finds regular expressions with dynamic components that are all constants"

Reply:
Thanks 👍 Updated.